### PR TITLE
merge strings for train 57094

### PR DESCRIPTION
**Pull request by bug-mirror**

### CODE REVIEW

File: locale/en/LC_MESSAGES/server.po [hunk 1]
```diff
@@ -358,7 +360,51 @@ msgstr ""
...
+#: lib/senders/templates/subscriptionPaymentFailed.html:3 lib/senders/templates/subscriptionPaymentFailed.txt:3
+msgid "It may be that your credit card has expired, or your current payment method is out of date."
+msgstr ""
+
+#: lib/senders/templates/subscriptionPaymentFailed.html:4
+msgid "Help us fix it by <a href=\"%(updateBillingUrl)s\">updating your payment information</a> within three days."
+msgstr ""
+
+#: lib/senders/templates/subscriptionPaymentFailed.html:5 lib/senders/templates/subscriptionPaymentFailed.txt:5
+msgid "Unfortunately, if we don’t hear from you within three days you’ll lose access to %(productName)s."
```
Comment (peiying2) [inline locale/en/LC_MESSAGES/server.po:396]: add a comma after `... within three days, ...`

**Comment by peiying2** [comment]
Screen Shot 2020-04-10 at 9 38 18 AM
This part is not newly added,  why I see both msgid and msgstr have content when the msgstr in`en` should be empty.

**Comment by peiying2** [comment]
@clouserw sorry another conflict again.  Let me know if the double entry (msgid and msgstr) is normal so I won't comment on that in the future. Thanks!

**Comment by clouserw** [comment]
I think having msgid and msgstr filled in for en is fine

**Comment by clouserw** [comment]
Punctuation change @ https://github.com/mozilla/fxa/pull/4905

**Comment by clouserw** [comment]
This will re-extract on Friday.

**Pull request closed without merging**